src/fsearch_list_view.c

```diff
@@ -100,9 +100,9 @@ struct _FsearchListView {
 };
 
 enum {
-    VIRTUAL_ROW_ABOVE_VIEW = -1,
```

Comment:
You can't remove this initialisation, they have been intialised as -1,-2 for a reason.
It is used as a sentinel index, example, see: https://github.com/cboxdoerfer/fsearch/blob/51da504585b0e566e5f33d4c87618559ba1921b2/src/fsearch_list_view.c#L679

Reply:
The other enums you've changed in this PR which are initialized to zero, it is ok to remove them.

Reply:
@porridgewithraisins I just reordered the three enum variants:

```diff
 enum {
-    VIRTUAL_ROW_ABOVE_VIEW = -1,
-    VIRTUAL_ROW_BELOW_VIEW = -2,
     UNSET_ROW = -3,
+    VIRTUAL_ROW_BELOW_VIEW,
+    VIRTUAL_ROW_ABOVE_VIEW,
 };


```

Reply:
Ah ok got it. My bad.